docs/moment/00-use-it/10-typescript.md

```diff
@@ -5,4 +5,33 @@ version: 2.13.0
 
 As of version **2.13.0**, Moment includes a typescript definition file.
 
-Examples of usages with typescript coming in the future.
\ No newline at end of file
+Install via NPM
+```
+npm install moment
+```
+
+Import and use in your Typescript file
+```javascript
+import * as moment from 'moment';
+
+
+let now = moment().format('LLLL');
+```
+
+**Note:** If you have trouble importing moment, try add ```"allowSyntheticDefaultImports": true``` in ```compilerOptions``` in your ```tsconfig.json``` file.
+
+**Locale Import**
+
+To use `moment.locale` you need first to import the language you are aiming for otherwise it'll keep in the current language.

```

Comment:
Minor grammatical error. Can we say:

To use `moment.locale` you first need to import the language you are targeting.